src/state/store.js

```diff
@@ -76,12 +72,7 @@ export const store = createStore(getDefaultState({ basedOnQueryParams: true }));
 
 store.reset = () => {
   store.reinitialize({
-    auth: store.getState().auth,
-    probeName: '',
-    probe: {
-      name: '',
-      loaded: false,
-    },
```

Comment:
I didn't see this earlier, but this shouldn't be necessary; the `reinitialize` method restores all of their values to their defaults, *except* things specified here. I refactored this a bit to make it more clear.